### ISSUE TITLE
Precision of reference intervals

### ISSUE THREAD

**Issue by Fasjul** [tuneinsight/glowing-bear-medco#45]
The float numbers should display only 2 (optionnally 3?) significant numbers using scientific notation.

This means, that the CI should have an equal (or bigger) number of significant digits. So if we display 2 significant digits for the RI, then the CI's should be display with 2 or preferably 3.

<img width="300" alt="168605542-65e6b5af-d345-43df-b9be-76ea520c1506" src="https://user-images.githubusercontent.com/11035984/182383265-8f67abd7-e7b9-4787-b7b8-be7e03884ed4.png">

**Comment by T-Blatter** [comment]
Rounding to significant numbers for displaying it could be done in JS, an approach has been proposed [here](https://stackoverflow.com/questions/56782846/significant-figures-in-javascript)

**Comment by romainbou** [comment]
Thanks Tobias.

I confirm that the latest implementation was using `toFixed(2)`. I pushed a fix in #65 that uses, like in your example, `toPrecision` with the agreed 5.

Some examples:
```
> 100.100.toFixed(2)
'100.10'
> 100.10001.toFixed(2)
'100.10'
> 0.0001.toFixed(2)
'0.00'
> parseFloat(100.100.toPrecision(5))
100.1
> parseFloat(100.10001.toPrecision(5))
100.1
> parseFloat(0.0001.toPrecision(5))
0.0001
```

Hence, a typical result would look like this:
![Screenshot from 2022-09-13 22-55-01](https://user-images.githubusercontent.com/725345/190007390-a21b48c1-8b8d-4c68-9ca5-9ef9dcb292bf.png)
